koku/api/iam/test/tests_views.py

```diff
@@ -85,9 +85,9 @@ def test_get_user_list(self):
         self.assertIsNotNone(results)
         self.assertEqual(len(results), 2)
         self.assertEqual(results[0]['username'],
-                         self.user_data[0]['username'])
-        self.assertEqual(results[1]['username'],
                          self.user_data[1]['username'])
+        self.assertEqual(results[1]['username'],
+                         self.user_data[0]['username'])
```

Comment:
The ordering is likely db-specific. So, it probably makes sense to collect the usernames into a list, and use assertIn() or a similar order-agnostic test.

Reply:
Order should be controlled by the following update:
https://github.com/project-koku/koku/pull/126/commits/d7d586d199c9fb2a9dc5ba9c99eb39f4dfc2a602

I just didn't change the test to reflect ordering by name as I should have.